pages/search.js

```diff
@@ -97,7 +97,9 @@ const FilterLabel = styled.label`
 const constructSortByQuery = sortByValue => {
   let query = {};
   if (sortByValue === 'ACTIVITY') {
-    query = { field: 'ACTIVITY', direction: 'DESC' };
+    query = { field: 'ACTIVITY', direction: 'ASC' };
```

Comment:
Looks like a terrible regression.

Reply:
Ops, sorry about that. 🤯 Somehow I missed this one.